File: src/main/java/com/wafflestudio/draft/model/enums/RoomStatus.java
package com.wafflestudio.draft.model.enums;

public enum RoomStatus {
    WAITING("waiting"),
    PLAYING("playing"),
    FINISHED("finished"),
    CLOSED("closed");

    private String value;

    RoomStatus(String value) {
        this.value = value;
    }

    public String getKey() {
        return name();
    }

    public String getValue() {
        return value;
    }
}

File: src/main/kotlin/com/wafflestudio/draft/model/enums/RoomStatus.kt
package com.wafflestudio.draft.model.enums

enum class RoomStatus(private val value: String) {
    WAITING("waiting"), PLAYING("playing"), FINISHED("finished"), CLOSED("closed");

    fun getKey(): String {
        return name
    }

    fun getValue(): String {
        return value
    }

}